src/lang/wasm.ts

```diff
@@ -230,7 +244,7 @@ export const parse = (code: string | Error): ParseResult | Error => {
     return new KCLError(
       parsed.kind,
       parsed.msg,
-      firstSourceRange(parsed),
+      bestSourceRange(parsed),
```

Comment:
why not iterate over them and show multiple i think this is what we used to do?

Reply:
i thought there was a playwright test and everything

Reply:
ah yeah thats how it was here https://github.com/KittyCAD/modeling-app/pull/2428

Reply:
Because that would display 20 errors, one for each call depth, when there's only 1 error.

Reply:
We support non-fatal errors now, so it's possible to have multiple real errors.

Reply:
oh LOL nevermind!!